Hippo.Core/Data/DbInitializer.cs

```diff
@@ -24,22 +24,56 @@ public async Task Initialize(bool recreateDb)
                 //do what needs to be done?
             }
 
-            var user = new User
-            {
+            await CheckAndCreateUser(new User {
                 Email = "jsylvestre@ucdavis.edu",
                 Kerberos = "jsylvest",
                 FirstName = "Jason",
                 LastName = "Sylvestre",
                 Iam = "1000009309",
-            };
+            });
+            await CheckAndCreateUser(new User
+            {
+                Email = "srkirkland@ucdavis.edu",
+                Kerberos = "postit",
+                FirstName = "Scott",
+                LastName = "Kirkland",
+                Iam = "1000029584",
+            });
+            await _dbContext.SaveChangesAsync();
+
+
+            if (!(await _dbContext.Accounts.AnyAsync()))
+            {
+                var ownerId = (await _dbContext.Users.FirstAsync(a => a.Iam == "1000029584")).Id;
+                var account = new Account()
+                {
+                    CanSponsor = true,
+                    OwnerId = ownerId,
+                };
+                await _dbContext.Accounts.AddAsync(account);
+                await _dbContext.SaveChangesAsync();

```

Comment:
Not that it matters but you don't need to save changes here, you can do it all at the end.  They way you would do it would be to set Sponsor to `account` instead of setting SponsorId via the re-query.   This is just dbInit code so I don't really care about this one but if it was anywhere else that's what i'd suggest